### PR TITLE
#1163 not all Flysystem adapters have the method applyPathPrefix

### PR DESCRIPTION
**Pull request by pieterw2w**

### CODE REVIEW

**Comment by garak** [comment]
Can you provide some context?

**Comment by pieterw2w** [comment]
This fixes issue #1163

File: src/Storage/FlysystemStorage.php [hunk 1]
```diff
@@ -67,7 +67,12 @@ protected function doResolvePath(PropertyMapping $mapping, ?string $dir, string
             return $path;
         }
 
-        return (string) $fs->getAdapter()->applyPathPrefix($path);
+        $adapter = $fs->getAdapter();
+        if (is_callable([$adapter, 'applyPathPrefix'])) {
```
Comment (garak) [inline src/Storage/FlysystemStorage.php:71]: please add native function invocation.
Ideally, you should run php-cs-fixer on your core before committing it.

**Review by garak — CHANGES_REQUESTED** [review]

**Comment by angelomelonas** [comment]
@pieterw2w thanks so much for this fix!

**Comment by garak** [comment]
Any news from this?

**Comment by pieterw2w** [comment]
> Any news from this?

sorry, i completely forgot about this PR. I'll see if I can fix these changes.

**Comment by pieterw2w** [comment]
Done!

**Review by garak — APPROVED** [review]

**Comment by garak** [comment]
Released in tag `1.16.1`.
Thank you!

**Pull request merged**